source/usage-examples.txt

```diff
@@ -26,7 +26,7 @@ Usage Examples
    Delete </usage-examples/delete-operations>
    Count Documents </usage-examples/count>
    Distinct Field Values </usage-examples/distinct>
-   Run a Command </usage-examples/command>
+   Run a Command Example </usage-examples/command>
```

Comment:
```suggestion
   Run a Command </usage-examples/command>
```
I would leave Example out of the TOC title to make the title concise. In the TOC, the page is already in the Usage Example section, so users can use that context to know the page is an example. Let me know if this make sense to you, thanks!